watem/watem/Calculate_Uparea.cpp

```diff
@@ -64,17 +64,17 @@ CCalculate_Uparea::CCalculate_Uparea()
 
     Parameters.Add_Value (
         NULL, "PCTOCROP", "Parcel Connectivity to cropland (%)",
-        "Percentage of water that will go from a cropland to another cropland.", PARAMETER_TYPE_Double, 70, 0, 100
+        "Percentage of water that will be reduced from a cropland to another cropland.", PARAMETER_TYPE_Double, 70, 0, 100
```

Comment:
Not clear if this is from cropland pixel to cropland pixel or parcel to parcel. It should be parcels, right? For the other statements this is more evident because the Landuse changes and this is clearly parcel based, however going from cropland to cropland could also be interpreted as within a parcel.

Reply:
i added parcels in brackets